bitcoinology/history-of-bitcoin-in-gaming.md

```diff
@@ -0,0 +1,211 @@
+
+Audience Respone: Can you put that back in because I've been trying to use my Sarutobi card. 
+
+Christian: Yeah, well, I'll probably get into like what happened, like why it kind of stopped. So yeah, I did have this at one point and yeah, I plan to bring it back soon. We have something cool we're working on at Zebedee, which I can't talk about because it's all Hush Hush secret, but yeah we do have an a like there's a lot of things wrong with NFTs and there's a lot of ways you should and shouldn't do it but it's ever D this is maybe I'll come back when once that's public and give a talk on that but yeah at some point it will come back But this is like really early stuff. It's kind of like, it reminds me of, I don't know if it's true, but I heard Blockbuster tried to make Netflix like in 1999 or something, or very early on. It was like, just didn't work. The tech wasn't there and there's so many things wrong. But now Netflix really works, right? And Blockbuster's gone. So I kind of see this stuff, you know, it's like so early. It was kind of cool, but it wasn't really the best way to do it. The tech wasn't there. It was a bit early. But yeah, hold on to that Sartutobi card, I suppose. Maybe it'll be useful. 
+
+## Scaling wars - Fee surge
+
+Christian: Yeah, anyway, so not so fast. So everything was great. We had Bitcoin games. People were tipping and dropping Bitcoin everywhere people playing Minecraft We had tokens game items and what could go wrong, right? You know, everything's great Yeah, and this happened fees scaling So probably a lot of people have probably got into Bitcoin after the scaling debate. But Bitcoin had a civil war where people were building applications on Bitcoin. But it didn't really scale. I don't want to get into the whole scaling debate, but loosely, to keep a blockchain decentralized, you have to have a limit on the amount of traffic you can do. The whole thing about Bitcoin is it should be reasonable for people to run their own node and reasonable for miners to kind of mine. But if there's too much traffic on the Bitcoin blockchain it centralizes that because it becomes too expensive to do so. Anyway, there was a big debate and we had this term small blockers and big blockers. And I've got a few images here. So this is when I guess I started to make my game and fees were zero pretty much. And then once my game started to become popular, in order to like, in order to send somebody a 10 cent tip, I had to pay $5 or £5, right? So my game mechanics broke and people who were trading the counterparty assets, people would trade them on a DEX where you could basically sell the image really good. But then once it became like a few dollars to kind of post a bid on the deck. It kind of killed it because you would try to sell a rare Pepe that was worth $5 and it would cost $1 to list it and then it might not get picked up. So it kind of, kind of died in a way. I'm gonna mention this guy. 
```

Comment:
```suggestion
Christian: Yeah, anyway, so not so fast. So everything was great. We had Bitcoin games. People were tipping and dropping Bitcoin everywhere people playing Minecraft. We had tokens game items and what could go wrong, right? You know, everything's great Yeah, and this happened fees scaling So probably a lot of people have probably got into Bitcoin after the scaling debate. But Bitcoin had a civil war where people were building applications on Bitcoin. But it didn't really scale. I don't want to get into the whole scaling debate, but loosely, to keep a blockchain decentralized, you have to have a limit on the amount of traffic you can do. The whole thing about Bitcoin is it should be reasonable for people to run their own node and reasonable for miners to kind of mine. But if there's too much traffic on the Bitcoin blockchain it centralizes that because it becomes too expensive to do so. Anyway, there was a big debate and we had this term small blockers and big blockers. And I've got a few images here. So this is when I guess I started to make my game and fees were zero pretty much. And then once my game started to become popular, in order to like, in order to send somebody a 10 cent tip, I had to pay $5 or £5, right? So my game mechanics broke and people who were trading the counterparty assets, people would trade them on a DEX where you could basically sell the image really good. But then once it became like a few dollars to kind of post a bid on the deck. It kind of killed it because you would try to sell a rare Pepe that was worth $5 and it would cost $1 to list it and then it might not get picked up. So it kind of, kind of died in a way. I'm gonna mention this guy. 
```